### PR TITLE
[Content] Updated NLCD Dataset PR

### PR DESCRIPTION
**Pull request by acblackford**
## PR Checklists
(Select the checklist that applies to your PR)

### Adding a Dataset Overview:
- [x] Dataset is available in the production VEDA STAC
- [x] Production STAC url is referenced in the dataset overview
- [x] All metadata, including those for the E&A page, is completed in the mdx file
- [x] Any images are compressed, ideally <500 KB
- [x] At least one code owner has reviewed the PR

### Adding a Data Story:
- [ ] Any data referenced in the story is already published to VEDA STAC and referenced in a separate dataset overview page (prior to merging or as part of the same PR)
- [ ] All metadata is completed in the mdx file
- [ ] Any images are compressed, ideally <500 KB
- [ ] Published Date is as close to the actual release date as possible (work with veda-ui team to determine potential release date when story is close to ready)
- [ ] At least one code owner has reviewed the PR

### Updating VEDA-UI
- [ ] Confirmed that [updating the `veda-ui` submodule](https://github.com/NASA-IMPACT/veda-config/blob/main/docs/DEVELOPMENT.md#development) is needed.
- [ ] Ensured that the correct version of veda-ui is being used
- [ ] Reviewed the veda-ui changelogs for relevant updates
- [ ] Tested that the changes work as expected with the current instance configuration

### CODE REVIEW

**Comment by acblackford** [comment]
@aboydnw @anayeaye I made a new branch and PR for the NLCD datasets, and everything seems to be working here now. If we can get this approved and released along with the black belt data story this week that would be great!

File: datasets/nlcd.data.mdx [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+      - Biomass
+      - Environmental Justice
+      - Land Cover
+  - name: Source
+    values:
+      - MRLC
+layers:
+  - id: nlcd-annual-conus
+    stacApiEndpoint: https://openveda.cloud/api/stac
+    tileApiEndpoint: https://openveda.cloud/api/raster
```
Comment (anayeaye) [inline datasets/nlcd.data.mdx:25]:
```suggestion
```

File: datasets/nlcd.data.mdx [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+        }
+    info:
+      source: MRLC
+      spatialExtent: United States
+      temporalResolution: Bi- to Tri-Annual 
+      unit: N/A 
+
+  - id: nlcd-new-urbanization
+    stacApiEndpoint: https://openveda.cloud/api/stac
+    tileApiEndpoint: https://openveda.cloud/api/raster
```
Comment (anayeaye) [inline datasets/nlcd.data.mdx:102]:
```suggestion
```

**Review by anayeaye — APPROVED** [review]

**Review by aboydnw — APPROVED** [review]
Looks good to me. @AliceR @dzole0311 sorry to slide this in after the prod PR was already made. Is it possible to add this to https://github.com/NASA-IMPACT/veda-config/pull/491 ?

**Comment by aboydnw** [comment]
@acblackford if this is ready, can you merge?

@dzole0311 let's include this when we bump veda-ui version this week

**Pull request merged**